integration

```diff
@@ -762,6 +762,7 @@
   "Sese-Schneider/ha-personio",
   "seven-io/home-assistant",
   "sfstar/hass-victron",
+  "sfstar/hass-heatapp_local",
```

Comment:
Looks abandoned/unfinished, no point of adding it here as the only issue that has ever been created in the repo has gone without response for months.
<https://github.com/sfstar/hass-heatapp_local/issues>